README.md

```diff
@@ -11,24 +11,66 @@ This library is inspired by the [PHP library PDFMerger](https://github.com/myoky
 ## Code sample
 
 ```javascript
-const PDFMerger = require('pdf-merger-js');
+const PDFMerger = require("pdf-merger-js");
 
 var merger = new PDFMerger();
 
 (async () => {
-  merger.add('pdf1.pdf');  //merge all pages. parameter is the path to file and filename.
-  merger.add('pdf2.pdf', [2]); // merge only page 2
-  merger.add('pdf2.pdf', [1, 3]); // merge the pages 1 and 3
-  merger.add('pdf2.pdf', '4, 7, 8'); // merge the pages 4, 7 and 8
-  merger.add('pdf3.pdf', '1 to 2'); //merge pages 1 to 2
-  merger.add('pdf3.pdf', '3-4'); //merge pages 3 to 4
-
-  await merger.save('merged.pdf'); //save under given name and reset the internal document
+  merger.add("pdf1.pdf"); //merge all pages. parameter is the path to file and filename.
+  merger.add("pdf2.pdf", [2]); // merge only page 2
+  merger.add("pdf2.pdf", [1, 3]); // merge the pages 1 and 3
+  merger.add("pdf2.pdf", "4, 7, 8"); // merge the pages 4, 7 and 8
+  merger.add("pdf3.pdf", "1 to 2"); //merge pages 1 to 2
+  merger.add("pdf3.pdf", "3-4"); //merge pages 3 to 4
+
+  await merger.save("merged.pdf"); //save under given name and reset the internal document
 })();
 ```
 
+### Browser Sample - React
```

Comment:
I would prefer an example without a custom ui-library like react, just plain vanilla javascript.
If you don't want to change this we could do this also in a  seperate pull-request!?

Reply:
Ok, let's get this merged and see how we go from there!

Reply:
Awesome. I'll work on a vanilla example and put in a PR to replace this.